lib/apiservers/engine/backends/volume.go

```diff
@@ -15,32 +15,79 @@
 package vicbackends
 
 import (
+	"encoding/json"
 	"fmt"
 	"net/http"
+	"os"
 
 	derr "github.com/docker/docker/errors"
 
 	"github.com/docker/engine-api/types"
 	"github.com/vmware/vic/lib/apiservers/portlayer/client/storage"
+	"github.com/vmware/vic/lib/apiservers/portlayer/models"
+	"github.com/vmware/vic/lib/guest"
 	"github.com/vmware/vic/pkg/trace"
 )
 
+//Volume : struct which defines the docker personalities view of a Volume
 type Volume struct {
 	ProductName string
 }
 
+type volumeMetadata struct {
+	Driver string
+	Name   string
+	Labels map[string]string
+}
+
+//Volumes : docker personality implementation for VIC
 func (v *Volume) Volumes(filter string) ([]*types.Volume, []string, error) {
 	return nil, make([]string, 0), fmt.Errorf("%s does not implement volume.Volumes", v.ProductName)
```

Comment:
That is something left over from the beginning of the file. I do not know for certain if the client will crash or report something nonsensical. Since I did not add this code, I  avoided speculating on it for now. There are no integration tests around it at the moment either.